### PR TITLE
Split

### PR DESCRIPTION
**Pull request by jonenst**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -250,16 +250,16 @@ $ INGRESS_HOST=`minikube ip`
 $ echo $INGRESS_HOST
 ```
 
-Fill config files with the INGRESS_HOST in k8s/overlays/local/ :
+Fill config files with the INGRESS_HOST in k8s/live/local/ :
 ```bash
-$ sed -i -e "s/<INGRESS_HOST>/${INGRESS_HOST}/g" k8s/overlays/local/*
+$ sed -i -e "s/<INGRESS_HOST>/${INGRESS_HOST}/g" k8s/live/local/*
```
Comment (bendaamerahm) [inline README.md:255]: dosent work cause there's a sub dir patches, maby we change it by using this cmd : 
`$ find k8s/live/local -type f -exec sed -i -e "s/<INGRESS_HOST>/${INGRESS_HOST}/g" {} \;`

File: README.md [hunk 2]
```diff
@@ -250,16 +250,16 @@ $ INGRESS_HOST=`minikube ip`
...
 ```
 
 Optionally, give an ssh access to the case importing cronjobs by providing your username and your password (or create a dedicated user for this if you want):
 The import jobs connect through ssh to your machine and automatically import files from the $HOME/opde and $HOME/boundaries
 ```bash
-$ sed -i -e 's/<USERNAME>/YOURUSERNAME/g' k8s/overlays/local/*
-$ sed -i -e 's/<PASSWORD>/YOURPASSWORD/g' k8s/overlays/local/*
+$ sed -i -e 's/<USERNAME>/YOURUSERNAME/g' k8s/live/local/*
+$ sed -i -e 's/<PASSWORD>/YOURPASSWORD/g' k8s/live/local/*
```
Comment (bendaamerahm) [inline README.md:262]: same

File: README.md [hunk 3]
```diff
@@ -270,7 +270,7 @@ $ docker-compose up -d
 
 Deploy k8s services:
 ```bash 
-$ kubectl apply -k k8s/overlays/local
+$ kubectl apply -k k8s/live/local
```
Comment (bendaamerahm) [inline README.md:273]: same

**Review by bendaamerahm — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jonenst) [reply README.md]: done

**Review by bendaamerahm — APPROVED** [review]

**Review by sBouzols — APPROVED** [review]

**Pull request merged**